templates/industrial/index.html

```diff
@@ -0,0 +1,695 @@
+      <p>Loved by developers and endorsed by sysadmins, Ubuntu Server is the starting point for technology innovation. Use the biggest open source software repository to build software-defined industrial assets.</p>
+      <a href="/server">
+        Learn more&nbsp;&rsaquo;
+      </a>
+    </div>
+    <div class="col-6">
+    <!-- TODO find a way to make the spacing right-->
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/9d42b2ab-3B-embedded-linux_AW.svg",
+          alt="",
+          height="150",
+          width="263",
+          hi_def=True,
+        ) | safe
+      }}
+      <h2 class="p-heading--four">Ubuntu Core</h2>
+      <p>Ubuntu Core is an ultra-secure embedded Ubuntu, optimised for mission critical IoT appliances. Benefit from OTA software updates and security maintenance for 10 years.</p>
+      <a href="/core">
+        Learn more&nbsp;&rsaquo;
+      </a>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--suru">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h2>ODM-certified</h2>     
+      <blockquote class="p-pull-quote">
+        <p class="p-pull-quote__quote">Canonical&rsquo;s commitment to ten years of support empowers us to commercialise business critical devices with the assurances our customers expect.</p>
+        <cite class="p-pull-quote__citation">Advantech</cite>
+      </blockquote>
+    </div>
+    <div class="col-4 u-vertically-center u-align--center u-hide--small" style="background: white;">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/1f876369-advantech.svg",
+          alt="Advantech",
+          height="33",
+          width="163",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "p-inline-images__logo"},
+        ) | safe
+      }}
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/185a14f4-Avnet.png",
+          alt="Avnet",
+          height="31",
+          width="163",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+  </div>
+  <!-- TODO find white separator -->
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
+
+    <div class="row u-equal-height">
+    <div class="col-8">
+      <h2>Silicon-endorsed</h2>     
+      <blockquote class="p-pull-quote">
+        <p class="p-pull-quote__quote">Intel and Canonical partner to accelerate  OEM readiness to meet growing demands for optimised sector-specific embedded solutions on Ubuntu Core, with long-term support and maintenance guaranteed.</p>
+        <cite class="p-pull-quote__citation">John Healey<br/>VP in the Internet of Things Group and GM of Platform Management and Customer Engineering.</cite>
+      </blockquote>
+    </div>
+    <!-- TODO fix those images -->
+    <div class="col-4 u-vertically-center u-align--center u-hide--small" style="background: white;">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/d8293409-intel_logo.svg",
```

Comment:
@spencerbygraves this logo has whitepace around it. can you provide one without please